auditors/deprecatedapis/depreceatedapis_test.go

```diff
@@ -53,6 +54,12 @@ func TestAuditDeprecatedAPIs(t *testing.T) {
 			require.Nil(t, err)
 			report := test.AuditManifest(t, fixtureDir, tc.file, auditor, []string{DeprecatedAPIUsed})
 			assertReport(t, report, tc.expectedSeverity, message, metadata)
+
+			// disable local tests when running in dev mode
+			if os.Getenv("USE_KIND") == "false" {
+				return
```

Comment:
in the end, [we decided on checking if the report is not nil](https://github.com/Shopify/kubeaudit/pull/453#discussion_r938090757), right? If we want to skip tests in this other part of the code, I'd open a separate PR since this is not necessarily part of the SARIF change. wdyt?